src/RaygunAspireWebApp/Views/Home/Index.cshtml

```diff
@@ -2,52 +2,8 @@
     ViewData["Title"] = "Error list";
 }
 
-@model List<ErrorInstanceRow>
+@model ErrorListViewModel
 
-<div class="background--color-bg-surface-01 box-shadow--light-elevation-02 border-color--color-border-elevation border-width--1 border-style--solid margin--24 padding-top--24">
-  <table id="errorTable" class="width--full">
```

Comment:
Note to reviewer: This was moved to the _ErrorList partial in order to be able to reload the table when clicking the "Load more" button that I've added (also found in the _ErrorList partial).